web/src/app/components/share-survey/share-survey.component.html

```diff
@@ -44,4 +40,16 @@
       <ground-general-access-control></ground-general-access-control>
     </mat-card-content>
   </mat-card>
+
+  <mat-card>
+    <ground-data-visibility-control class="floating-button"></ground-data-visibility-control>
+
+    <mat-card-header>
+      <mat-card-title>Data Visibility</mat-card-title>
```

Comment:
Please use "Sentence case", not "Title Case" for consistency.
https://uxdesign.cc/title-case-vs-sentence-case-in-ux-writing-212087192261